week4:JSweek2/2.js

```diff
@@ -0,0 +1,11 @@
+var Imawesome = 'I`m awesome';
+
+console.log(Imawesome);
+
+/* 
+console.log('I'm awesome');
+
+this code did not have any value above,
+and using ' caused problems. wrong format
```

Comment:
Do you remember using backticks ``?
For example you could use:
````
console.log(`...`);
````

Reply:
I see what I did wrong.